index.html

```diff
@@ -43,7 +43,7 @@
           >
         </li>
         <li>
-          <a href="#how-to-help" class="header__link header__link_border"
+          <a href="#donations" class="header__link header__link_border"
```

Comment:
Do we use id in BEM?

Reply:
That's not an id, that's a marker for a local link.

Reply:
You can use href="#". I don't think you need the  id="donations" and the name after the "#"

Reply:
Ok, though that would leave the link not working. Wouldn't it be more logical if the "Help Us" link took us to the donations part of the site?

Reply:
On second thought, you are right. Leave it as it is.

Reply:
👍